CONTRIBUTING.rst

```diff
@@ -277,57 +295,16 @@ detailing the test coverage::
 Building docs
 -------------
 
-To build the HTML documentation, you can run:
+To build the HTML documentation, run:
 
 .. code:: sh
 
     spin docs
 
-Then, all the HTML files will be generated in ``scikit-image/doc/build/html/``.
-To rebuild a full clean documentation, run:
-
-.. code:: sh
-
-    spin docs --clean
-
-Requirements
-~~~~~~~~~~~~
-
-`Sphinx <http://www.sphinx-doc.org/en/stable/>`_,
-`Sphinx-Gallery <https://sphinx-gallery.github.io>`_,
-and LaTeX are needed to build the documentation.
-
-**Sphinx:**
-
-Sphinx and other python packages needed to build the documentation
-can be installed using: ``scikit-image/requirements/docs.txt`` file.
-
-.. code:: sh
-
-    pip install -r requirements/docs.txt
-
-.. _sphinx_gallery:
-
-**Sphinx-Gallery:**
-
-The above install command includes the installation of
-`Sphinx-Gallery <https://sphinx-gallery.github.io>`_, which we use to create
-the :ref:`examples_gallery`.
-Refer to the Sphinx-Gallery documentation for complete instructions on syntax and usage.
-
-If you are contributing an example to the gallery or editing an existing one,
-build the docs (see above) and open a web browser to check how your edits
-render at ``scikit-image/doc/build/html/auto_examples/``: navigate to the file
-you have added or changed.
-
-When adding an example, visit also
-``scikit-image/doc/build/html/auto_examples/index.html`` to check how the new
-thumbnail renders on the gallery's homepage. To change the thumbnail image,
-please refer to `this section
-<https://sphinx-gallery.github.io/stable/configuration.html#choosing-thumbnail>`_
-of the Sphinx-Gallery docs.
+Output is in ``scikit-image/doc/build/html/``.  Add the ``--clean``
+flag to build from scratch, deleting any cached output.
 
-Note that gallery examples should have a maximum figure width of 8 inches.
+You may need the following system requirements:
```

Comment:
Curious, how is LaTeX used when building the HTML for our docs?

Reply:
Good question! Maybe it used to be for formulas; do we do mathjax now? If so, get rid of it!